extensions/positron-python/python_files/posit/positron/data_explorer.py

```diff
@@ -1321,11 +1327,22 @@ def _get_type(cls, column, column_index, state: DataExplorerState):
         if dtype == object:  # noqa: E721
             type_name = cls._get_inferred_dtype(column, column_index, state)
             type_name = cls.TYPE_NAME_MAPPING.get(type_name, type_name)
+            type_display = cls._get_type_display(type_name)
+        elif isinstance(dtype, pd.CategoricalDtype):
+            type_name = "category"
```

Comment:
nit: should this be `categorical` to match the pandas name of the dtype? I do like brevity though

Reply:
"category" is what `str(dtype)` returns so I will replace this hard-coded string with that for clarity.